src/sparc/client/client.py

```diff
@@ -69,39 +67,52 @@ def __init__(self, config_file="config/config.ini", connect=True) -> None:
                 f"{__package__}.services.{module_name}", config[current_config], connect
             )
 
-    def add_module(self, path, config, connect=True):
+    #        else:
+    #            self.add_module(package_dir, config[current_config], connect)
+
+    def add_module(self, paths, config=None, connect=True):
         """Adds and optionally connects to a module in a given path with configuration variables defined in config.
 
-        Parameters
-        ----------
-        path : str
+        Parameters:
+        -----------
+        paths : str
```

Comment:
Following from my comment above: this comment is already outdated since I can see from the code that you also accept ``list[str]``

Reply:
yeap